PhilanthropyDataCommons/front-end
# Implement proposal detail page UI changes

_With gratitude, this PR replaces #630. I realized while reviewing @hminsky2002's efficient work that some larger refactoring was justified, as well as a feature I _didn't_ mention in the original issue: moving Data Provider buttons to the org page rather than removing it altogether._

This PR implements the desired UI changes for the proposal page now that we have proper organization pages.

- Sidebar and header are now proposal-centric instead of org-centric
- Data Providers button (Candid / Charity Navigator) moved to org page
- `ListGridItem` component added

**Testing:**
1. On `main`, go to any proposal page (e.g. http://localhost:3000/proposals/464)
2. Note the contents of the sidebar and panel header
3. Switch to this branch and reload that page
4. Note that the sidebar and panel header are quite different! (See #601 for the specific differences.) Rather than elevating organization-related fields in order to make the page look like an org page, we now elevate proposal-specific fields (name or summary). We've also moved the Data Providers button to the org page. Speaking of which…
5. Go to http://localhost:3000/organizations. Assuming you've added some, if you load an org page, you'll see the Data Providers button in the top right of the panel.

Closes #602
